### ISSUE TITLE
Adding a login page

### ISSUE THREAD

**Issue by tharunbirla** [tharunbirla/kollab#3]
Creating a frontend login page.

**Comment by iamrahulrnair** [comment]
hey, man i could work on this 😄

**Comment by tharunbirla** [comment]
Yeah go for it 👍

**Comment by iamrahulrnair** [comment]
could u reorganize the module structure, everything is bundled in the same src folder

**Comment by tharunbirla** [comment]
Yeah you can add `Components` folder for the components and `assets` folder for the css in src
